### PR TITLE
Linter: Assume normal severity if not given

### PR DESCRIPTION
**Pull request by sils1297**
CC @AbdealiJK for review

### CODE REVIEW

File: coalib/bearlib/abstractions/Linter.py [hunk 1]
```diff
@@ -239,14 +239,17 @@ def _convert_output_regex_match_to_result(self,
             # Pre process the groups
             groups = match.groupdict()
 
-            try:
-                groups["severity"] = severity_map[
-                    groups["severity"].lower()]
```
Comment (AbdealiJK) [inline coalib/bearlib/abstractions/Linter.py]:
```
 -           try:
 -                groups["severity"] = severity_map[
 -                    groups.get("severity", RESULT_SEVERITY.NORMAL).lower()]
```
is simpler ?

**Comment by AbdealiJK** [comment]
ack cdf95e9634e608e59f21af03650380830a8d646e
CHecked coverage 100% manually

**Comment by sils1297** [comment]
ack cdf95e9

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**